spring-cloud-dataflow-server/src/test/resources/application.properties

```diff
@@ -1,4 +1,4 @@
 spring.datasource.url=jdbc:h2:mem:testdb
-spring.cloud.dataflow.features.streams-enabled=false
-spring.autoconfigure.exclude=org.springframework.cloud.dataflow.rest.client.config.DataFlowClientAutoConfiguration
+spring.cloud.dataflow.client.skip-ssl-validation=true
+spring.cloud.dataflow.client.enable-dsl=true
```

Comment:
I see this property `enableDsl` is enabled here but I don't see any references of this property on how it is getting used in this code base?

Reply:
Not sure either. It comes from the spring-cloud-dataflow-rest-client. Maybe it is obsolete, as it doesn't seem to effect the behavior.

Reply:
Probably property when dsl was external lib.

Reply:
ok, maybe we can remove it now?